docs/organizations/invitations.mdx

```diff
@@ -0,0 +1,144 @@
+```
+
+Once the user visits the invitation link, they will be redirected to the page you specified, which means you must handle the authentication flow in your code for that page. See the [custom flow](/docs/organizations/accept-organization-invitations) guide for more information.
+
+> [!TIP]
+> For testing redirect URL's in your development environment, you can pass your port (`http://localhost:3000`). If you would like to use Clerk's Account Portal, you can pass your Clerk Frontend API URL as the base URL. For example, `https://prepared-phoenix-98.clerk.accounts.dev/sign-up` would send the user to the Account Portal sign-up page. You can find your Frontend API URL in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. On the left side, select **Show API URLs**.
+
+### Invitation metadata
+
+You can also add metadata to an invitation when creating the invitation through the Backend API. Once the invited user signs up using the invitation link, the invitation metadata (`OrganizationInvitation.public_metadata`) will end up in the user's metadata (`User.public_metadata`). You can find more information about user metadata in the [metadata](/docs/users/metadata) docs.
+
+To add metadata to an invitation, you can use the `public_metadata` property when the invitation is created.
+
+The following example demonstrates how to create an invitation with metadata using cURL.
+
+<SignedIn>
+  Replace the `<YOUR_ORGANIZATION_ID>` with the ID of the organization you want to invite the user to. Replace the `user_123` with the ID of the user who is inviting the other user. Replace the email address with the email address you want to invite. Your secret key is already injected into the code snippet.
+</SignedIn>
+
+<SignedOut>
+  Replace the `<YOUR_ORGANIZATION_ID>` with the ID of the organization you want to invite the user to. Replace the `user_123` with the ID of the user who is inviting the other user. Replace the email address with the email address you want to invite. Replace `YOUR_SECRET_KEY` with your Clerk secret key. You can find your secret key in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page.
+</SignedOut>
+
+```bash
+curl 'https://api.clerk.com/v1/organizations/<YOUR_ORGANIZATION_ID>/invitations' \
+  -X POST \
+  -H 'Authorization: Bearer {{secret}}' \
+  -H 'Content-Type: application/json' \
+  -d '{ "inviter_user_id": "user_123", "email_address": "test@gmail.com", "role": "org:member", "public_metadata": {"age": "21"} }'
+```
+
+## Revoke an invitation
+
+Revoking an invitation prevents the user from using the invitation link that was sent to them.
+
+At the moment, you can only revoke invitations via the [Backend API](/docs/reference/backend-api/tag/Organization-Invitations#operation/RevokeOrganizationInvitation){{ target: '_blank' }}.
+
```

Comment:
‼️ This is incorrect, FAPI and clerk-js allows this. 

End-users (admins of an org) can revoke any invitation that was created for that org.

Reply:
ahh then this is outdated information ! I see here that we do have a FAPI method: https://clerk.com/docs/references/javascript/organization-invitation#revoke

Reply:
check out this update: [db5563f](https://github.com/clerk/clerk-docs/pull/1509/commits/db5563f68720f972d1b5b9a5d8701d5505f7115f)